src/components/ManageUsersModal.jsx

```diff
@@ -42,6 +43,19 @@ const styles = (theme) => ({
   openButton: {
     width: '100%',
   },
+  iconButton: {
+    border: `1px solid ${theme.palette.primary.main}`,
+    borderRadius: '3px',
+    padding: '1px',
+    backgroundColor: theme.palette.primary.main,
```

Comment:
Is this not standard button styling?

Reply:
I think if you give it the primary variant, the normal button styling should work

Reply:
sorry, `contained` variant